src/vs/platform/actionWidget/browser/actionList.ts

```diff
@@ -307,11 +308,25 @@ export class ActionList<T> extends Disposable {
 		}
 	}
 
+	private onFocus() {
+		this._list.domFocus();
+		const focused = this._list.getFocus();
+		if (focused.length === 0) {
+			return;
+		}
+
+		const focusIndex = focused[0];
+		const element = this._list.element(focusIndex);
+		this._delegate.onFocus?.(element.item);
+
```

Comment:
[nit] Make sure to clean up some of the extra newlines added throughout the changes